pro/src/routes/OfferEducationalStockCreation/OfferEducationalStockCreation.tsx

```diff
@@ -44,18 +37,20 @@ const OfferEducationalStockCreation = (): JSX.Element => {
   }
 
   useEffect(() => {
-    const loadOffer = async () => {
-      const { payload, message, isOk } = await getOfferAdapter(offerId)
-      if (isOk) {
-        setOffer(payload.offer)
-        setIsReady(true)
-        return
+    if (!isReady) {
+      const loadOffer = async () => {
+        const { payload, message, isOk } = await getOfferAdapter(offerId)
+        if (isOk) {
+          setOffer(payload.offer)
+          setIsReady(true)
+          return
+        }
+        notify.error(message)
```

Comment:
préférence personnelle, mais bon : 
```suggestion
        if (!isOk) {
          return notify.error(message)
        }
        
        setOffer(payload.offer)
        setIsReady(true)
```